core/src/main/scala/sttp/tapir/testing/EndpointVerificationError.scala

```diff
@@ -0,0 +1,50 @@
+package sttp.tapir.testing
+
+import sttp.tapir.AnyEndpoint
+
+class EndpointVerificationError {
+  def showAllPaths(e: AnyEndpoint) = {
```

Comment:
can be private?

Reply:
also, what does this function do? :)

Reply:
Moved to companion object. 
Function returns method with concatenated final path from security input and standard input.